### ISSUE TITLE
resolvedModule of undefined, due to API change in TypeScript 1.6.2?

### ISSUE THREAD

**Issue by Coderah** [TypeStrong/typedoc#148]
receiving this error

```
Using TypeScript 1.6.2 from /home/coderah/code/moss-piglet/node_modules/typescript/lib
undefined:29636
            : (function (moduleNames, containingFile) { return ts.map(moduleNames, function (moduleName) { return resolveModuleName(moduleName, containingFile, options, host).resolvedModule; }); });
                                                                                                                                                                              ^

TypeError: Cannot read property 'resolvedModule' of undefined
    at eval (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29636:175)
    at Object.map (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:155:29)
    at resolveModuleNamesWorker (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29636:67)
    at processImportedModules (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:30013:35)
    at findSourceFile (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29975:21)
    at processSourceFile (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29916:27)
    at processRootFile (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29863:13)
    at eval (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29649:60)
    at Object.forEach (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:90:30)
    at Object.createProgram (eval at <anonymous> (/home/coderah/code/moss-piglet/node_modules/typedoc/bin/typedoc.js:41:3), <anonymous>:29649:16)
```

unsure if this is due to the breaking API change in 1.6.2 documented here https://github.com/Microsoft/TypeScript/wiki/API-Breaking-Changes#compilerhost-interface-change-comparing-to-typescript-16-beta any thoughts?

**Comment by sebastian-lenz** [comment]
Hi Alex, nice to hear from you. The stack shows that the error is triggered within the TypeScript compiler (line 41 is the point we [load TypeScript](https://github.com/sebastian-lenz/typedoc/blob/master/bin/typedoc.js#L37-L41)). Did you run your project against TypeScript 1.6.2 and does it work? If that succeeds, did you ensure to pass the exact same compiler options to TypeDoc as you did to TypeScript?

**Comment by sebilasse** [comment]
merely an assumption:
I had the same issue with `resolvedType` in my own project and this became 'internal' in the recent version of the compiler API.
However resolvedModule is 
/*
         * Denotes if 'resolvedFileName' is isExternalLibraryImport and thus should be proper external module:
         * - be a .d.ts file 
         * - use top level imports\exports
         * - don't use tripleslash references
         */

**Comment by aciccarello** [comment]
I'm wondering if this was fixed with #455. We can reopen if I'm wrong.
